cli/internal/client/client.go

```diff
@@ -40,17 +39,33 @@ type ApiClient struct {
 // ErrTooManyFailures is returned from remote cache API methods after `maxRemoteFailCount` errors have occurred
 var ErrTooManyFailures = errors.New("skipping HTTP Request, too many failures have occurred")
 
+// _maxRemoteFailCount is the number of failed requests before we stop trying to upload/download
+// artifacts to the remote cache
+const _maxRemoteFailCount = uint64(3)
```

Comment:
I'm embarrassed to admit that I just learned that "unsigned int" has nothing to do with cryptography 😬